### ISSUE TITLE
Verbage

### ISSUE THREAD

**Issue by joomlaidiots** [joomla/joomla-cms#40753]
Whomever was the genius behind the naming of your main components needs to be fired. My first time having to traverse your system will hopefully be the last. In 40 years of programming I have never seen such absurd names for the components of a web site. Article for page? really? On the first  'article' of the Administrators guide you use the word article 3 different ways describing different things. No-one over there saw a problem?  I'm sorry for venting but I just spent 6 hours trying to fix one typo on a customers article container.

And I fail to see the need for the so-called CMS systems.  A novice could have been half way through a primer on asp in the time it took me to muddle through to find the right 'article' html that was in a database!! html code in a database??

### Steps to reproduce the issue
1. 

### Device and system info

* What version of Joomla were you using?

* What device were you using? (eg, phone, laptop)

* What web browser were you using? (eg, Chrome, Safari)

* If you were using assistive technology, please let us know the device or software you were using (eg VoiceOver, JAWS, sip-and-puff device, etc)

### Additional comments

If you know which [WCAG](https://www.w3.org/TR/WCAG/) or [ATAG](https://www.w3.org/TR/ATAG20/) criteria this violates, please indicate it here:

**Comment by zero-24** [comment]
Ok thanks for your feedback. Do you have a specific question or can we close this issue here?
